zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -143,7 +143,7 @@ pub(crate) const CONSENSUS_BRANCH_IDS: &[(NetworkUpgrade, ConsensusBranchId)] =
     (Blossom, ConsensusBranchId(0x2bb40e60)),
     (Heartwood, ConsensusBranchId(0xf5b9230b)),
     (Canopy, ConsensusBranchId(0xe9ff75a6)),
-    (Nu5, ConsensusBranchId(0x37519621)),
+    (Nu5, ConsensusBranchId(0xc2d6d0b4)),
```

Comment:
Source:
https://github.com/zcash/zcash/pull/5741/files#diff-5b867b4ded1f0001e04eafaab7d257969d76138c9e5eabd73640fd9fbfd37cb2R48

The ZIP will eventually be updated:
https://zips.z.cash/zip-0252#nu5-deployment

Reply:
Here is the ZIP PR:
https://github.com/zcash/zips/pull/600/files#diff-b424a91f01c7799201470397654fcf582c045c318b0f91e1e15af4b7fdfc8f78